book_version/04-simulation.Rmd

```diff
@@ -55,19 +55,19 @@ print(pi_hat)
 
 ## Finite Sample Evaluation
 
-Asymptotic theory is apparatus to approximate finite sample distributions.
+Asymptotic theory is apparatus to approximate infinite sample distributions.
 In the real world, nothing is infinite, so apparently all asymptotic results are fake.
 However, asymptotic theory is so far the best possible instrument that helps us
 understand the behavior of estimators and tests, either in econometrics or in statistics in general.
 Simulation is one way to evaluate the accuracy of approximation.
 
 Even though real data empirical example can also be used to illustrate a statistical procedure,
-artificial data is convenient and boast advantages. The prevalent paradigm in statistics is
-to assume that the data are generated from a model. We, as researchers, check how close is the estimate to
+artificial data is convenient and boasts advantages. The prevalent paradigm in statistics is
```

Comment:
The correct one should be "artificial data are convenient and boast advantages". "data" is plural.